dex/pair/src/liquidity_pool.rs

```diff
@@ -189,6 +225,16 @@ pub trait LiquidityPoolModule:
         self.calculate_k_constant(&first_token_amount, &second_token_amount)
     }
 
+    fn calculate_k_for_virtual_reserves(&self, token_side_id: &TokenIdentifier) -> BigUint {
```

Comment:
these are functions from front-running-protection branch which was merged into main. We do not need these yet.